pkg/syncer/admissionhandler/admissionhandler.go

```diff
@@ -139,7 +139,8 @@ func StartWebhookServer(ctx context.Context) error {
 			return err
 		}
 	}
-	if containerOrchestratorUtility.IsFSSEnabled(ctx, common.CSIMigration) {
+	if containerOrchestratorUtility.IsFSSEnabled(ctx, common.CSIMigration) ||
+		containerOrchestratorUtility.IsFSSEnabled(ctx, common.BlockVolumeSnapshot) {
```

Comment:
There are different implications based on which fss is enabled, would we want to validate storage class if the csi migration is disabled but block volume snapshot is enabled? IMO there must different `validationHandler` based on fss, it may get messy, but seems like the right thing to do.

Reply:
> There are different implications based on which fss is enabled, would we want to validate storage class if the csi migration is disabled but block volume snapshot is enabled? IMO there must different `validationHandler` based on fss, it may get messy, but seems like the right thing to do.

Inside each `validationHandler`, the FSS is checked again. I think we are good.